### PR TITLE
feat: Add manifest, icons, build script for Microsoft Edge store package

### PR DESCRIPTION
**Pull request by peterdur**
#### Description of changes

To enable publishing Accessibility Insights for Web to the extension store for Microsoft Edge Insider, create an APPX manifest XML file and add supporting store icons. Also add a PowerShell script that uses the Windows SDK MakeAppx tool to generate the APPX file to publish to the store.

#### Pull request checklist

- [X] Addresses an existing issue: Required for #606 
- [X] Added relevant unit test for your changes. (`npm run test`) 
    - not applicable, no code changes
- [X] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage` - not applicable, no code changes
    - not applicable, no code changes
- [X] Ran precheckin (`npm run precheckin`)
- [X] (UI changes only) Added screenshots/GIFs to description above
    - not applicable, no UI changes
- [X] (UI changes only) Verified usability with NVDA/JAWS
    - not applicable, no UI changes

### CODE REVIEW

File: AppXManifest.xml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (smoralesd) [inline AppXManifest.xml]: I'm guessing we actually need the xml version & encoding as first tag in this file (as opposed to the license header)

File: new-appx.ps1 [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+if (Test-Path ./drop/productionEdge) { Remove-Item ./drop/productionEdge -Recurse }
+if (Test-Path ./drop/productionEdge.appx) { Remove-Item ./drop/productionEdge.appx }
+Copy-Item ./drop/production ./drop/productionEdge -Recurse
+Copy-Item ./AppXManifest.xml ./drop/productionEdge/AppXManifest.xml
+&"c:\Program Files (x86)\Windows Kits\10\bin\10.0.17134.0\x64\makeappx.exe" pack /h SHA256 /d .\drop\productionEdge /p .\drop\productionEdge.appx
```
Comment (smoralesd) [inline new-appx.ps1]: Is `10.0.17134.0` a version number (on the path we're using)
Seems a little brittle to have a "changing" version on the path.

File: AppXManifest.xml (hunk 1 above)
Comment (peterdur) [reply AppXManifest.xml]: Yup, that's correct. The XML fails validation in MakeAppx if the comment precedes the XML header.

File: new-appx.ps1 (hunk 2 above)
Comment (peterdur) [reply new-appx.ps1]: Fair question. 10.0.17134.0 is the Windows SDK version for the April 2018 update. The SDK install, at least on my machine, looks to be additive - old versions appear to be retained as new ones are added (my box has folders for the 5 most recent versions of Windows 10). So I don't expect this version to need to change during the time window that we'll be using this script.

**Pull request merged**